src/store/ChartStore.js

```diff
@@ -268,6 +245,33 @@ class ChartStore {
         stxx.append('createDataSet', this.updateComparisons);
     }
 
+    @action.bound changeSymbol(symbolObj) {
```

Comment:
Check against current symbol, do not re-render if it's the same symbol.

Reply:
This check is performed in the dropdown (https://github.com/binary-com/SmartCharts/blob/dev/src/store/ChartTitleStore.js#L34) when the symbols are selected, but adding the check here to me is a good safety measure. I'll add this as well.